### PR TITLE
EVG-14664 Update hostspawn tooltip to leafy green

### PR DESCRIPTION
**Pull request by bynn**
image
image
image
image

### CODE REVIEW

File: src/pages/spawn/spawnHost/EditSpawnHostButton.tsx [hunk 1]
```diff
@@ -18,35 +17,45 @@ export const EditSpawnHostButton: React.FC<EditSpawnHostButtonProps> = ({
...
-          >
-            <span>{children}</span>
-          </Tooltip>
-        )}
+      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+      <div
+        onClick={(e: React.MouseEvent) => e.stopPropagation()}
+        onMouseEnter={() => {
+          setOpenTooltip(true);
+          console.log(openTooltip);
```
Comment (SupaJoon) [inline src/pages/spawn/spawnHost/EditSpawnHostButton.tsx:28]: left over log

Comment (bynn) [reply src/pages/spawn/spawnHost/EditSpawnHostButton.tsx]: done

**Review by SupaJoon — CHANGES_REQUESTED** [review]

**Comment by SupaJoon** [comment]
your e2e test failure was an io issue and all test specs passed. i restarted the e2e task to see if it happens again

**Review by SupaJoon — APPROVED** [review]

File: src/pages/spawn/spawnHost/EditSpawnHostModal.tsx [hunk 2]
```diff
@@ -179,38 +180,51 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
...
-              )}
+            {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
+            <div
+              onMouseEnter={() => {
+                setOpenTooltip(true);
+              }}
+              onMouseLeave={() => {
+                setOpenTooltip(false);
+              }}
             >
```
Comment (khelif96) [inline src/pages/spawn/spawnHost/EditSpawnHostModal.tsx:191]: Same comment here

**Review by khelif96 — CHANGES_REQUESTED** [review]

**Comment by SupaJoon** [comment]
created this ticket to address the failing test https://jira.mongodb.org/browse/EVG-14742

File: src/pages/spawn/spawnHost/EditSpawnHostButton.tsx [hunk 3]
```diff
@@ -18,35 +17,44 @@ export const EditSpawnHostButton: React.FC<EditSpawnHostButtonProps> = ({
...
+                  name: "Open the Edit Spawn Host Modal",
+                  hostId: host.id,
+                  status: host.status,
+                });
+              }}
+            >
+              Edit
+            </PaddedButton>
+          }
         >
```
[the remark this answers is not in the document]
Comment (bynn) [reply src/pages/spawn/spawnHost/EditSpawnHostButton.tsx]: done

File: src/pages/spawn/spawnHost/SpawnHostButton.tsx [hunk 4]
```diff
@@ -34,27 +34,43 @@ export const SpawnHostButton = () => {
...
+          }
         >
-          Spawn a host
-        </PlusButton>
-      </Tooltip>
+          {reachedMaxNumHosts
+            ? `You have reached the maximum number of hosts (${maxHosts}). Delete some hosts to spawn more.`
+            : undefined}
+        </Tooltip>
+      </span>
```
[the remark this answers is not in the document]
Comment (bynn) [reply src/pages/spawn/spawnHost/SpawnHostButton.tsx]: done

File: src/pages/spawn/spawnHost/EditSpawnHostModal.tsx (hunk 2 above)
Comment (bynn) [reply src/pages/spawn/spawnHost/EditSpawnHostModal.tsx]: done

**Review by khelif96 — APPROVED** [review]
Great Job LGTM!

**Pull request merged**